data-es2016plus.js

```diff
@@ -368,6 +368,101 @@ exports.tests = [
     ],
   },
   {
+    name: 'import()',
+    categpry: '2017 features',
+    significance: 'small',
+    spec: 'https://tc39.github.io/proposal-dynamic-import/',
+    subtests: [
+      {
+        name: 'import',
+        exec: function () {/*
+          try {
+            return typeof import === "function";
```

Comment:
this is false; `import()` is strictly **not** a function, it's a keyword followed by parens.